src/draggable-list/tests/unit/DraggableList.spec.tsx

```diff
@@ -0,0 +1,209 @@
+const { describe, it, afterEach, beforeEach } = intern.getInterface('bdd');
+const { assert } = intern.getPlugin('chai');
+
+import { create, tsx } from '@dojo/framework/core/vdom';
+import { renderer, assertion, wrap } from '@dojo/framework/testing/renderer';
+import DraggableList, { move } from '../../index';
+import * as themeCss from '../../../theme/default/draggable-list.m.css';
+import GlobalEvent from '../../../global-event';
+import drag from '@dojo/framework/core/middleware/drag';
+import { SinonStub, stub } from 'sinon';
+import global from '@dojo/framework/shim/global';
+
+const middlewareFactory = create();
+let isDragging = false;
+let start: {} | undefined = undefined;
+let addEventListenerStub: SinonStub;
+
+if (!global.document) {
```

Comment:
Might want to wrap setting `global` properties in a `beforeEach` and set it back on an `afterEach` to avoid potential confusion with other tests